source/patterns/02-organisms/site-header/site-header.twig

```diff
@@ -10,7 +10,7 @@
       {%- set image = image|default({})|merge({src: image.src|default(asset('images/libero-logo.svg')), alt: image.alt|default('Site logo'), class: image.class|default([])|merge(['site-header__image'])}) -%}
 
       {%- if href is defined -%}
-        <a href="{{ href }}">
+        <a href="{{ href }}" class="site-header__logo_link">
```

Comment:
Don't think attributes on this needs configuring, so directly applying `class` in this case.